### ISSUE TITLE
make usage help more regular

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#859]
Currently, the usage help looks like:
```
usage: borg mount [-h] [--critical] [--error] [--warning] [-v] [--debug]
                  [--lock-wait N] [--show-rc] [--no-files-cache] [--umask M]
                  [--remote-path PATH] [-f] [-o OPTIONS]
                  REPOSITORY_OR_ARCHIVE MOUNTPOINT
```
The `-v` is somehow a irregularity as the other loglevel setting options are shown in the long form, just that one is short (and also it does not fit into the log level sequence critical > error > warning > info > debug).

Could look like that, if we change the order of alternatives, so `--info` is first:
```
usage: borg mount [-h] [--critical] [--error] [--warning] [--info] [--debug]
                  [--lock-wait N] [--show-rc] [--no-files-cache] [--umask M]
                  [--remote-path PATH] [-f] [-o OPTIONS]
                  REPOSITORY_OR_ARCHIVE MOUNTPOINT
```

`-v` and `--verbose` would be still available as documented alternative to `--info`, just not listed there.

**Comment by enkore** [comment]
Or we replace that part just with "<options>", because most commands have so many (and even the basic options all commands have already span two-and-a-half lines!), that I doubt anyone reads that.

Also it would be good if the options can be grouped in the block below, e.g. logging options, common options (lock-wait, show-rc, no-files-cache, umask, remote-path, ...) and command options separately.

For reference, the rather cluttered and unstructered output of "borg create --help":

```
usage: borg create [-h] [-v] [--debug] [--lock-wait N] [--show-version]
                   [--show-rc] [--no-files-cache] [--umask M]
                   [--remote-path PATH] [-s] [-p] [--list]
                   [--filter STATUSCHARS] [-e PATTERN]
                   [--exclude-from EXCLUDEFILE] [--exclude-caches]
                   [--exclude-if-present FILENAME] [--keep-tag-files]
                   [-c SECONDS] [-x] [--numeric-owner]
                   [--timestamp yyyy-mm-ddThh:mm:ss]
                   [--chunker-params CHUNK_MIN_EXP,CHUNK_MAX_EXP,HASH_MASK_BITS,HASH_WINDOW_SIZE]
                   [--ignore-inode] [-C COMPRESSION] [--read-special] [-n]
                   ARCHIVE PATH [PATH ...]

Create new archive

positional arguments:
  ARCHIVE               name of archive to create (must be also a valid
                        directory name)
  PATH                  paths to archive

optional arguments:
  -h, --help            show this help message and exit
  -v, --verbose, --info
                        enable informative (verbose) output, work on log level
                        INFO
  --debug               enable debug output, work on log level DEBUG
  --lock-wait N         wait for the lock, but max. N seconds (default: 1).
  --show-version        show/log the borg version
  --show-rc             show/log the return code (rc)
  --no-files-cache      do not load/update the file metadata cache used to
                        detect unchanged files
  --umask M             set umask to M (local and remote, default: 0077)
  --remote-path PATH    set remote path to executable (default: "borg")
  -s, --stats           print statistics for the created archive
  -p, --progress        show progress display while creating the archive,
                        showing Original, Compressed and Deduplicated sizes,
                        followed by the Number of files seen and the path
                        being processed, default: False
  --list                output verbose list of items (files, dirs, ...)
  --filter STATUSCHARS  only display items with the given status characters
  -e PATTERN, --exclude PATTERN
                        exclude paths matching PATTERN
  --exclude-from EXCLUDEFILE
                        read exclude patterns from EXCLUDEFILE, one per line
  --exclude-caches      exclude directories that contain a CACHEDIR.TAG file
                        (http://www.brynosaurus.com/cachedir/spec.html)
  --exclude-if-present FILENAME
                        exclude directories that contain the specified file
  --keep-tag-files      keep tag files of excluded caches/directories
  -c SECONDS, --checkpoint-interval SECONDS
                        write checkpoint every SECONDS seconds (Default: 300)
  -x, --one-file-system
                        stay in same file system, do not cross mount points
  --numeric-owner       only store numeric user and group identifiers
  --timestamp yyyy-mm-ddThh:mm:ss
                        manually specify the archive creation date/time (UTC).
                        alternatively, give a reference file/directory.
  --chunker-params CHUNK_MIN_EXP,CHUNK_MAX_EXP,HASH_MASK_BITS,HASH_WINDOW_SIZE
                        specify the chunker parameters. default: 19,23,21,4095
  --ignore-inode        ignore inode data in the file metadata cache used to
                        detect unchanged files.
  -C COMPRESSION, --compression COMPRESSION
                        select compression algorithm (and level): none == no
                        compression (default), lz4 == lz4, zlib == zlib
                        (default level 6), zlib,0 .. zlib,9 == zlib (with
                        level 0..9), lzma == lzma (default level 6), lzma,0 ..
                        lzma,9 == lzma (with level 0..9).
  --read-special        open and read special files as if they were regular
                        files
  -n, --dry-run         do not create a backup archive
```

**Comment by ThomasWaldmann** [comment]
The stuff is auto-generated, so doing bigger changes would be more effort, if possible.

**Comment by ThomasWaldmann** [comment]
fixed in 1.0-maint, soon in master.
